src/recursive_sorting/recursive_sorting.py

```diff
@@ -1,17 +1,39 @@
-# TO-DO: complete the helpe function below to merge 2 sorted arrays
+# TO-DO: complete the help function below to merge 2 sorted arrays
+a=[3,6,9]
+b=[2,4,8]
 def merge( arrA, arrB ):
     elements = len( arrA ) + len( arrB )
-    merged_arr = [0] * elements
+    merged_arr = []
     # TO-DO
-    
-    return merged_arr
+    i = j = 0
+    while (i < len(arrA) and j < len(arrB)):
+        if arrA[i] >= arrB[j]:
+            merged_arr.append(arrB[j])
+            j+=1
+        else:
+            merged_arr.append(arrA[i])
+            i+=1
+    merged_arr += arrA[i:]
+    merged_arr += arrB[j:]
 
+    return merged_arr
+    
+print(merge(a, b))
 
 # TO-DO: implement the Merge Sort function below USING RECURSION
+array = [2, 1, 4, 53, 62, 7, 12, 9]
 def merge_sort( arr ):
     # TO-DO
+    if len(arr) <= 1:
+        return arr
+    cut = int(len(arr)/2)
```

Comment:
there is nothing wrong with this, but i figured i would share in case you don't know of it yet:

alternatively you could have `cut = len(arr) // 2`  the double `//` operator in python takes the floor of the division result (an integer) as well.